src/cdk/drag-drop/drag-ref.ts

```diff
@@ -1244,11 +1244,14 @@ export class DragRef<T = any> {
       this._initialTransform = this._rootElement.style.transform || '';
     }
 
-    // Preserve the previous `transform` value, if there was one. Note that we apply our own
-    // transform before the user's, because things like rotation can affect which direction
-    // the element will be translated towards.
-    this._rootElement.style.transform = this._initialTransform ?
-      transform + ' ' + this._initialTransform  : transform;
+    // Preserve the previous `transform` value, only if is there is one and not 'none'.
+    // 'none' is valid css transform value but none + translate3d(x, y, z) is invalid.
+    // Note that we apply our own transform before the user's, because things like
+    // rotation can affect which direction the element will be translated towards.
+    this._rootElement.style.transform = this._initialTransform
```

Comment:
Rather than checking here, it might be easier to check inside `_applyRootElementTransform` before saving the value to `_initialTransform`. That way we don't have to account for it again if we add any new places that read from `_initialTransform`.